presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -239,16 +239,17 @@ public BaseJdbcClient(
                 ResultSet resultSet = getColumns(tableHandle, connection.getMetaData())) {
             List<JdbcColumnHandle> columns = new ArrayList<>();
             while (resultSet.next()) {
+                String columnName = resultSet.getString("COLUMN_NAME");
                 JdbcTypeHandle typeHandle = new JdbcTypeHandle(
                         resultSet.getInt("DATA_TYPE"),
                         Optional.ofNullable(resultSet.getString("TYPE_NAME")),
                         resultSet.getInt("COLUMN_SIZE"),
                         resultSet.getInt("DECIMAL_DIGITS"),
                         Optional.empty());
                 Optional<ColumnMapping> columnMapping = toPrestoType(session, connection, typeHandle);
+                log.debug("Mapping data type of '%s' column '%s': %s mapped to %s", tableHandle, columnName, typeHandle, columnMapping);
```

Comment:
I don't think you want to log the full table handle here.

It's probably better to do a single debug log at the end of the loop, where we can log a nicely formatted column list with the mappings.